.gitignore

```diff
@@ -30,3 +30,5 @@
 # Build directory
 build
 
+nbproject
```

Comment:
These are present in my repository.  I use netbeans and netbeans automatically creates nbproject.  I also create the build directory within the project directory.  This is why I added them to .gitignore.